src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -30,6 +32,7 @@ class JwtManager(object):
     JWT_TOKEN_TTL = 28800  # default 8 hours
     JWT_ALGORITHM = 'HS256'
     _secret = None
+    oauth2_token = False
```

Comment:
Is this a per-user or a server global setting? If you don't know why I'm asking this look closely at the line underneath.

Reply:
It was a global setting. I have removed it and using the mgr.SSO_DB now